src/yacs-web/src/components/ScheduleEvent.vue

```diff
@@ -22,19 +22,19 @@ export default {
 }
 </script>
 
-<style scoped lang="scss">
```

Comment:
Is there a reason for removing the `scoped`?

Reply:
I'm guessing there's something weird going on between the inline styles and the scoped styles but removing it makes it look correct